lib/models/ui/comment_post.dart

```diff
@@ -1,13 +1,16 @@
+import "package:cloud_firestore/cloud_firestore.dart";
```

Comment:
The `models/ui` part of the app should not depend on `clound_firestore`. We should use the `DateTime` type instead of the `Timestamp` type here and call `toDate` at the creation of the model class for the UI (at the interface between database and UI models to stay consistent).

**Note**: this also holds for `PostOverview`.

Reply:
fixed in b68bd5c65184f7aaf40ce9ecf79f64acd4dbd84f